index.html

```diff
@@ -288,14 +288,15 @@ <h2 class="text-lg text-gray-900 font-medium title-font mb-4 px-4">
           class="h-full flex items-center border-gray-200 border p-4 rounded-lg"
         >
           <img
-            alt="team"
+            alt="Şule KOÇ"
             class="w-16 h-16 bg-gray-100 object-cover object-center flex-shrink-0 rounded-full mr-4"
-            src="https://randomuser.me/api/portraits/women/89.jpg"
+            src="images/crew/sule-koc.png"
```

Comment:
```suggestion
            src="/images/crew/sule-koc.png"
```

Bunu tüm resimler için değiştirsek iyi olur.